src/serverMain/kotlin/com/studo/campusqr/Server.kt

```diff
@@ -126,8 +126,8 @@ suspend fun main() {
         }
       }
       route("report") {
-        post("list") { call.getAuthenticatedCall()?.returnReportData() }
-        post("listActiveCheckIns") { call.getAuthenticatedCall()?.listAllActiveCheckIns() }
+        get("list") { call.getAuthenticatedCall()?.returnReportData() }
```

Comment:
this endpoint is used in the moderation frontend so you have to adjust the XHR call there.